_schemaV2.graphql

```diff
@@ -11267,7 +11267,7 @@ type Money {
   ): Float!
 
   # An amount of money expressed in minor units (like cents).
-  minor: Int!
+  minor: Long!
```

Comment:
Is this considered a breaking change? I know that our code/resolvers might be safe, but I wonder if front-end clients will complain. Have you tried importing this newly generated schema into Force (for instance) locally, and trying to boot and run Force?

Reply:
I tested using Eigen. There was no issues. I will test with Force also.